taca/nanopore/minion.py

```diff
@@ -230,4 +230,13 @@ def dump_path(self):
         path_to_write = os.path.join(proj, sample, run)
         f = open(new_file, 'w')
         f.write(path_to_write)
-        f.close()
\ No newline at end of file
+        f.close()
+    
+    def write_finished_indicator(self):
+        """Write a hidden file to indicate 
+        when the finial rsync is finished."""
+        new_file = os.path.join(self.run_dir, '.sync_finished')
+        f = open(new_file, 'w')
+        f.write('0')
+        f.close()
```

Comment:
I think I prefer the "with open():" style, then "f.close()" is done automatically. This is perfectly fine though.

Reply:
I do too 😄 And turns out there is a touch function in python, which is even better since the file can be empty.

Reply:
⭐